static/js/containers/LoginButton.js

```diff
@@ -17,9 +17,10 @@ class LoginButton extends React.Component {
     </span>;
 
     return (
-      <DropdownButton
+      <SplitButton
         title={title}
         bsStyle="danger"
+        href="/dashboard"
```

Comment:
Let's try to avoid setting `href` here - doing so means we do a full page refresh instead of triggering a react-router route transition.

I think if you want to use the `SplitButton` component you may be able to either use the `LinkContainer` component (see below) or set up a quick `onClick` handler than pushes the correct url onto the react-router history.